server/webhook.go

```diff
@@ -390,17 +390,25 @@ func (s *Server) processFile(ctx context.Context, pushEvent vcs.PushEvent, repo
 	}
 
 	// Create schema update issue
+	issueName := fmt.Sprintf("%s by %s", migrationDescription, strings.TrimPrefix(file, repo.BaseDirectory+"/"))
+	if err := s.createIssueFromPushEvent(ctx, issueName, pushEvent, repo, migrationType, migrationDetailList); err != nil {
+		return "", false, activityCreateList, echo.NewHTTPError(http.StatusInternalServerError, "Failed to create issue").SetInternal(err)
+	}
+
+	return fmt.Sprintf("Created issue %q on adding %s", issueName, file), true, activityCreateList, nil
+}
+
+func (s *Server) createIssueFromPushEvent(ctx context.Context, issueName string, pushEvent vcs.PushEvent, repo *api.Repository, migrationType db.MigrationType, migrationDetailList []*api.MigrationDetail) error {
```

Comment:
Do we need to pass in the repo / migrationType? Shall we use migrationInfo and projectID instead?